src/utils/deepCopy.test.js

```diff
@@ -1,24 +1,15 @@
 import { deepCopy } from 'utils/deepCopy'
-import { isAccessor } from './dataTypeChecker'
 
 describe('Make a deeo copy of an object', () => {
   const input = {
     a: 'a',
     b: 'b',
-    c: [1, 2, 3, 4, 5],
-    get getA() {
-      return this.a
-    },
-    set setA(v) {
-      this.a = v
-    }
+    c: [1, 2, 3, 4, 5]
```

Comment:
This test could be improved by adding another array inside of c and then testing if their reference is the same.

```js
expect(output.c[output.c.length - 1]).not.toBe(input.c[input.c.length - 1])
```